libnavui-status/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="mapbox_status_view_background">#4D000000</color>
+    <color name="mapbox_status_view_foreground">@android:color/white</color>
```

Comment:
```suggestion
    <color name="mapbox_status_view_foreground">@color/colorOnSurface</color>
```
This will enable switching b/w day/night mode seamlessly. Also, we would like to keep using the colors defined in `libnavui-resources` to be consistent unless a really different color is required.

Reply:
👍  Which colors from `libnavui-resources` should I use for background and foreground?